src/ExtMsg.actor.cpp

```diff
@@ -249,8 +249,10 @@ ACTOR Future<Void> runCommand(Reference<ExtConnection> nmc,
 
 		if (e.code() == error_code_bad_dispatch) {
 			bob.append("errmsg", format("no such cmd: %s", cmd.c_str()));
+			bob.append("$err", format("no such cmd: %s", cmd.c_str()));
 		} else {
 			bob.append("errmsg", format("command [%s] failed with err: %s", cmd.c_str(), e.what()));
+			bob.append("$err", format("command [%s] failed with err: %s", cmd.c_str(), e.what()));
```

Comment:
There are many other places, in our code, we are returning only `errmsg`. Do we need to change there as well?

I was looking for a difference between `errmsg` and `$err` in the wire protocol. The only place I found any explanation was here https://github.com/mongodb/mongo-python-driver/blob/master/pymongo/helpers.py#L164. It says `errmsg` is present until 1.8. Does it mean all the new versions will return `$err` for all error messages?

Reply:
Yeah I am on that now.